src/mixins/dom.js

```diff
@@ -0,0 +1,34 @@
+// DomMixin
+//  ---------
+
+import Backbone from 'backbone';
+
+export default {
+  createBuffer() {
+    return document.createDocumentFragment();
+  },
+
+  appendChildren(el, children) {
+    Backbone.$(el).append(children);
+  },
+
+  beforeEl(el, sibling) {
+    Backbone.$(el).before(sibling);
+  },
+
+  replaceEl(newEl, oldEl) {
+    Backbone.$(oldEl).replaceWith(newEl);
```

Comment:
I believe we can change this back in v4 after we remove `preventDestroy` as people can just `detachView` before the empty I think..